pos_barcode_tare/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,12 @@
+This add-on enable POS to read and print tare barcodes. A tare barcode is used to sell loose goods in a bring your own container (BYOC) scheme.
+
+The BYOC scheme has four steps:
+    1. The cashier weights the container and sticks the tare bar code onto the customer's container.
+    2. The customer takes the desired quantity of the good s-he wants to buy.
+    3. The cashier weights the filled container and good, POS gives the corresponding price.
+    4. The cashier scans the tare bar code, POS removes the container's weight from the latest product of the order.
+
+This add-on adds a news screen to POS to (web) print the tare barcode labels. This add-on enables POS to read a tare
+barecode. When such a barcode is read, POS adjusts the weight of the latest article in the order list.
+The new weight is equal to the total weight minus the tare weight. The price is updated accordingly to the weight
+change.
```

Comment:
could you add a screenshot ? 

something like : 

    ..figure: ../static/description/point_of_sale_tare_screen.png

Reply:
Will d :)